packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -95,6 +104,14 @@ def imshow(
         position of the [0, 0] pixel of the image array, in the upper left or lower left
         corner. The convention 'upper' is typically used for matrices and images.
 
+    labels : dict with str keys and str values (default `{}`)
+        Overrides names used in the figure for axis titles (keys ``x`` and ``y``),
+        colorbar title (key ``colorbar``) and hover (key ``color``). The values
```

Comment:
to match other PX functions there should be no `colorbar` and `color` should be used there....

Reply:
I liked the idea of having the possibility of having different labels for the hover and the colorbar title...

Reply:
it can still be overriden of course but not in px land.

Reply:
ok, I compromised here :-)

Reply:
thank you :)